references/kvm-admin-api/test/integration/features/js-callout.feature

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Feature:
+  As an API platform operator
+  I want to be able to manipulate entries in a KVM
+  So that I can use it to store config values
+
+  Scenario: Create an entry in the KVM
+    Given I set body to {"value":"test-value"}
+    And I set Content-Type header to application/json
+    When I PUT /kvms/kvmap/entries/testentry
```

Comment:
General question - I would expect Creating a KVM entry to be a POST like it was in the old Management API. Put would be used for the change. This follows the RESTful convention for each resource

C - POST
R - GET
U - PUT/PATCH (depending on partial payload and idempotency)
D - DELETE

any reason why we didnt do this?